recipes/papillon/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "papillon" %}
+{% set version = "0.0.8" %}
+{% set sha256 = "7505f977ed053b9eff70ba3a6a0787b227f36c419d8d742e3c74d09ebc7bc9c5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  skip: True  # [py<35]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python --single-version-externally-managed --record record.txt
+    - setuptools
+    - pandas >=0.17.1
```

Comment:
`pandas` and `seaborn` do not look like they are required to `build` the package -- just as run time dependencies, so you can probably remove them from the `build` section.

Reply:
ok, thanks.